blockchain/common_test.go

```diff
@@ -170,7 +171,9 @@ func newFakeNode(parent *blockNode, blockVersion int32, stakeVersion uint32, bit
 		Nonce:        testNoncePrng.Uint32(),
 		StakeVersion: stakeVersion,
 	}
-	return newBlockNode(header, parent)
+	node := newBlockNode(header, parent)
+	node.status = statusValid | statusDataStored
```

Comment:
The order of these bit flags are opposite of line 133.

Reply:
Good call.  Updated.